ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -79,7 +79,7 @@ extension RACSignal {
 
 /// Turns each value into an Optional.
 private func optionalize<T, E>(signal: Signal<T, E>) -> Signal<T?, E> {
-	return signal |> map { Optional.Some($0) }
+	return signal |> map { Optional($0) }
```

Comment:
[here](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/f7611e4747816e9828ab8b48674a1e531c64c4d4/ReactiveCocoa/Swift/Bag.swift#L32) and [here](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/f7611e4747816e9828ab8b48674a1e531c64c4d4/ReactiveCocoa/Swift/Signal.swift#L531) use `Optional(value)`, so I unified with it. Should change the two instead?